m2cgen/assemblers/__init__.py

```diff
@@ -16,6 +16,7 @@
     # LightGBM
     "LGBMRegressor": LightGBMModelAssembler,
     "LGBMClassifier": LightGBMModelAssembler,
+    "Booster": LightGBMModelAssembler,
```

Comment:
I'm not sure this will work out nicely since `XGBoost` use the same class name for its Booster.

Reply:
This is another point which makes me question whether we really need this change. 

@chris-smith-zocdoc just to double check, all these changes are required to simply avoid 2 following lines of code:
```
estimator = lgb.LGBMClassifier()
estimator._Booster = booster
```

Is it correct?

Reply:
There are some cases where more properties need to be set 

If `base_score` is used, you'll also need to set `estimator.base_score`

For LGBMClassifier set `estimator.n_classes_`


Maybe it makes sense to have a function to convert Booster -> LGBMRegressor/LGBMClassifier instead?